docs/api/README.md

```diff
@@ -895,6 +896,36 @@ const debounce = (ms, pattern, task, ...args) => fork(function*() {
 ### `debounce(ms, channel, saga, ...args)`
 You can also handel a channel as argument and the behaviour is the same as [`debounce(ms, pattern, saga, ..args)`](#debouncems-pattern-saga-args)
 
+### `retry(maxTries, delay, fn, ...args)`
+Creates an Effect description that instructs the middleware to call the function `fn` with `args` as arguments while `fn` does not return the result of `fn` calls count less than `maxTries`. `Delay` instructs `retry` to make a pause between `fn` calls.
```

Comment:
we could simplify a bit
```
Creates an Effect description that instructs the middleware to call the function `fn` with `args` as arguments.
In case of failure will try to make another call after `delay` milliseconds, if number of attempts < `maxTries`
```
WDYT?

Reply:
Yeap, i think nice, ty.